src/models/db.js

```diff
@@ -82,6 +95,15 @@ export default config => {
     if ('associate' in db[modelName]) {
       db[modelName].associate(db);
     }
+
+    if (!config.hooks) {
+      return;
+    }
+
+    hooks.forEach(hook => {
+      const clientHook = config.hooks[hook];
+      clientHook && db[modelName].hook(hook, clientHook);
+    });
```

Comment:
It seems that we can create [global/universal hooks](http://docs.sequelizejs.com/en/latest/docs/hooks/#global-universal-hooks) \o/ So you don't need to do this for each model.